web/views.py

```
@@ -5242,6 +5243,167 @@ def toggle_course_status(request, slug):
+            nft_badge = mint_nft_badge(achievement_id, wallet_address, contract_address, admin_private_key)
+
+            if nft_badge:
+                messages.success(request, f"NFT badge successfully minted and sent to {wallet_address}!")
+                return redirect("achievement_detail", achievement_id=achievement_id)
+            else:
+                messages.error(request, "Failed to mint NFT badge. Please try again later.")
+        except Exception as e:
+            messages.error(request, f"Error minting NFT badge: {str(e)}")
+
+        return render(
+            request, "achievements/send_nft_badge.html", {"achievement": achievement, "student_wallet": student_wallet}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)